rotkehlchen/db/schema.py

```diff
@@ -816,6 +816,21 @@
 );
 """
 
+DB_CREATE_OPEN_NODES = """
+CREATE TABLE IF NOT EXISTS open_nodes(
```

Comment:
Why are you making a new DB Table for this? Wasn't this going in the settings anyway? Why put the hard coded nodes which you already have in a mapping the code also in the DB and in the schema itself?